docs-chef-io/content/server/upgrades.md

```diff
@@ -48,7 +48,13 @@ The Chef Infra Server 14 upgrade does not automatically reindex existing externa
 
 #### Upgrading to 14.8
 
-Chef Infra Server 14.8 upgrades PostgreSQL from 9.6 to 13.3. The 14.8 upgrade process requires a one-time downtime to vacuum, upgrade, and re-index the database. The entire upgrade operation takes approximately one minute per 1000 nodes. This process may take longer depending on your server hardware and the size of the node objects on your Chef Infra Server.
+Chef Infra Server 14.8 upgrades PostgreSQL from 9.6 to 13.3. The 14.8 upgrade process requires a one-time downtime to vacuum, upgrade, and re-index the database. The entire upgrade operation takes approximately one minute per 1000 nodes (1000 nodes will approximately be 286MB). This process may take longer depending on your server hardware and the size of the node objects on your Chef Infra Server.
+
+{{< note >}}
+
+The attribute `postgresql['pg_upgrade_timeout']` in [chef-server.rb]({{< relref "config_rb_server_optional_settings" >}}) is used to set the timeout value for pg upgrade. Set this value accordingly to the time required for your data size. 
```

Comment:
Should `postgresql['pg_upgrade_timeout']` be added to config_rb_server_optional_settings.md page? Cause I don't see it in there.

Reply:
Ya, that is needed. I have updated the docs. Thanks!